JulioC-19/LargeProjectMobile
# Julio/signup screen

This PR implements the functionality and style of the signup screen, type checking navigator to move between screens, some new packages and improved custom components. 
The avatar selection functionality is not implemented yet, so it is a static image for now, some discussion on this is required. 
Added color.js that contains our color pallet and assets folder for the images.
This PR defaults to a generic home screen with a button to navigate to the signup screen, once login is in, we'll connect the screens together. 

Testing Instructions: 
Run and build the app. 
Navigate to signup and enter values for each field, inspect that the keyboard does not block the fields, the screen should move up as the keyboard appears. 
The password field has a feature that shows the plain text input if the eye icon is tabbed. 
Tab sign up button, a prompt should appear with the user's first and last name and the fields should reset. 
Tab the Log in text to navigate back to the generic home screen.
